### ISSUE TITLE
feature request: close to tray, minimize to tray

### ISSUE THREAD

**Issue by bseymour1** [input-output-hk/daedalus#1009]
Would love the ability to leave daedalus running in the tray for when staking begins.

**Comment by zoehess** [comment]
Minimize to tray is super useful, especially if one is utilizing daedelus throughout the day.

**Comment by jeryd** [comment]
This would be really helpful. Also needs an option to "start to tray" so it never gets in the way of the the windows start up process.

**Comment by oo-00** [comment]
When I wrote this request nearly 3 years ago, I was under the misunderstanding that daedalus would need to be running in order to stake. I think it the feature would still be nice, but it's not as pertinent as I thought it was.

I see someone is assigned to work on it now. Thanks guys!

**Comment by LeoniePhiline** [comment]
I still think this would be a nice feature, and should be not too difficult with Electron.
Many apps do it right, like the Element (formerly Riot) IM app for the matrix chat network, which is also built on Electron.

**Comment by nikolaglumac** [comment]
Yes, we have it in our backlog and will focus on it as soon as we get some free bandwidth to do so...
Thanks for the patience!

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
